plugins/modules/site_workflow_manager.py

```diff
@@ -1971,7 +1971,7 @@ def main():
         ccc_site.reset_values()
         ccc_site.get_want(config).check_return_status()
         ccc_site.get_have(config).check_return_status()
-        ccc_site.upload_floor_image(config)
+        # ccc_site.upload_floor_image(config)
```

Comment:
if it is not required, then kindly remove it..

Reply:
It is required, I'II uncomment it